README.md

```diff
@@ -6,42 +6,45 @@ Golang SDK for interacting with the Wavefront v2 API, and sending metrics throug
 
 ### API Client
 
-Presently support for:
- * Querying
- * Searching
- * Dashboard Management
- * Alert (and Alert Target) Management
- * Events Management
-
-Please see the [examples](examples) directory for an example on how to use each, or check out the [documentation](https://godoc.org/github.com/WavefrontHQ/go-wavefront-management-api).
+Supports most public API paths. Refer to the
+[documentation](https://godoc.org/github.com/WavefrontHQ/go-wavefront-management-api)
+for further information.
 
 ```Go
 package main
 
 import (
-    "log"
+  "fmt"
+  "log"
+  "os"
 
-    wavefront "github.com/WavefrontHQ/go-wavefront-management-api"
+  wavefront "github.com/WavefrontHQ/go-wavefront-management-api"
 )
 
 func main() {
-    client, err := wavefront.NewClient{
-        &wavefront.Config{
-            Address: "test.wavefront.com",
-            Token:   "xxxx-xxxx-xxxx-xxxx-xxxx",
-        },
-    }
+  client, err := wavefront.NewClient(
+    &wavefront.Config{
+      Address: os.Getenv("WAVEFRONT_ADDRESS"),
+      Token:   os.Getenv("WAVEFRONT_TOKEN"),
+    },
+  )
 
-    query := client.NewQuery(
-        wavefront.NewQueryParams(`ts("cpu.load.1m.avg", dc=dc1)`),
-    )
+  query := client.NewQuery(
+    wavefront.NewQueryParams(`ts("cpu.load.1m.avg", dc=dc1)`),
+  )
 
-    if result, err := query.Execute(); err != nil {
-        log.Fatal(err)
-    }
+  result, err := query.Execute()
```

Comment:
Thank you for fixing the scope of result. I don't see how this would have ever worked before.